bin/tmux-nerd-font-window-name

```diff
@@ -2,13 +2,21 @@
 
 NAME=$1
 
+# Use reverse shell icon if option is true
+ALTERNATE_SHELL_ICON="$(tmux show -gqv '@tmux-nerd-font-window-name-alterate-shell-icon')"
+SHELL_ICON=""
+
+if [ "$ALTERNATE_SHELL_ICON" = true ]; then
```

Comment:
I prefer we use a function to scope the data.

```
get_shell_icon() {
	local default_shell_icon=''
	local shell_icon="$(tmux show -gqv '@tmux-nerd-font-window-name-shell-icon')"
	[ -n "$shell_icon" ] && echo "$shell_icon" || echo "$default_shell_icon"
}

SHELL_ICON=$(get_shell_icon)
```

Please test this and clean it up as you see fit.